dev/breeze/src/airflow_breeze/utils/ci_group.py

```diff
@@ -22,7 +22,7 @@
 
 
 @contextmanager
-def ci_group(title: str, enabled: bool = True):
+def ci_group(title: str, enabled: bool = True, message_type: str = "[info]"):
```

Comment:
Should the arg passed just be the type name rather than also require the square brackets?

```python
def ci_group(title: str, enabled: bool = True, message_type: str = "info"):
    ....

# Handle print formatting later
get_console().print(f"[{message_type}]{title}[/]")
```

Reply:
Right. Good point. Otherwise [/]  - might be wrong.

Reply:
Fixed it and actually even converted it into an Enum :)